src/createAnimatedComponent.js

```diff
@@ -344,6 +357,18 @@ export default function createAnimatedComponent(Component) {
       return props;
     }
 
+    _obtainViewName(forceComponent) {
```

Comment:
What is `forceComponent` – it does not seem like we don't pass that parameter anywhere

Reply:
fixed c1d13d9